addons/crm/models/crm_lead.py

```diff
@@ -672,6 +672,8 @@ def _get_duplicated_leads_by_emails(self, partner_id, email, include_lost=False)
         domain = partner_match_domain
         if not include_lost:
             domain += ['&', ('active', '=', True), ('probability', '<', 100)]
+        else:
+            domain += ['&', ('active', '!=', None), '|', ('type', '=', 'opportunity'), '&', ('type', '=', 'lead'), ('active', '=', True)]
```

Comment:
Active not none ? What are you trying to achieve here ? This question is rhetorical, by the way. I don't know why you are trying to set several leaves based on active, by the way.

By the way, reverse the condition to make it easier to understand.

Rewrite your condition actually.